lively.halos/morph.js

```diff
@@ -2117,6 +2117,10 @@ export default class Halo extends Morph {
     this.target.onHaloDoubleMouseDown(evt);
   }
 
+  onDoubleMouseDown (evt) {
```

Comment:
Where do we use this?

Reply:
I thought it was something that happened when I rebased on your branch/ branched from your branch... it is nothing that I wrote at least that I remember

Reply:
Hm shouldn´t be there, can you rebase again? Or just remove it here